data-plane/src/server/data_plane_server.rs

```diff
@@ -89,6 +94,24 @@ where
 
                 match req.parse(&buffer) {
                     Ok(Status::Complete(body_offset)) => {
+                        #[cfg(feature = "enclave")]
+                        if req.path == Some("/.well-known/attestation") {
+                            let response_bytes = match handle_attestation_request(req).await {
+                                Ok(response) => response_to_bytes(response).await,
+                                Err(err) => {
+                                    eprintln!("Failed to handle attestation request - {err:?}");
+                                    shutdown_conn(&mut stream).await;
+                                    break;
```

Comment:
I think it would be better here to build an internal server error response and return it so the client can parse and catch 5XX errors